casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -71,12 +71,13 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
         for {
           evalR                     <- newEval(deploy :: Nil, runtime, start)
           (_, Seq(processedDeploy)) = evalR
-          result <- if (processedDeploy.status.isFailed) Seq.empty[Datum[ListParWithRandom]].pure[F]
-                   else {
-                     val r: F[Seq[Datum[ListParWithRandom]]] =
-                       runtime.space.getData(name).map(_.toSeq)
-                     r
-                   }
+          result <- if (processedDeploy.status.isFailed)
+                      Sync[F].raiseError(new Exception("Evaluation error:" + processedDeploy.status))
+                    else {
+                      val r: F[Seq[Datum[ListParWithRandom]]] =
```

Comment:
Not related to your change but looks like this is a leftover from debugging and can be removed.